gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Copied from kotlinc
 org.gradle.jvmargs=-Duser.country=US -Dkotlin.daemon.jvm.options=-Xmx2200m -Dfile.encoding=UTF-8
 
-kotlinBaseVersion=1.4.10
-intellijVersion=193.6494.35
```

Comment:
Not sure if this is necessary, I have a commit without this change but still works.

Reply:
my tests were failing until i updated intellijVersion.

Reply:
i tried reverting it but then tests fail with:

```
'void com.intellij.core.CoreApplicationEnvironment.registerExtensionPointAndExtensions(java.nio.file.Path, java.lang.String, com.intellij.openapi.extensions.ExtensionsArea)'
java.lang.NoSuchMethodError: 'void com.intellij.core.CoreApplicationEnvironment.registerExtensionPointAndExtensions(java.nio.file.Path, java.lang.String, com.intellij.openapi.extensions.ExtensionsArea)'
	at org.jetbrains.kotlin.cli.jvm.compiler.CoreApplicationEnvironmentCompatKt.registerExtensionPointAndExtensionsEx(coreApplicationEnvironmentCompat.kt:17)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.registerApplicationExtensionPointsAndExtensionsFrom(KotlinCoreEnvironment.kt:537)
	
```

so your tests pass w/o this change?

Reply:
I see, my change is without test enabled due to test-mutes module issue, then it makes sense to upgrade intellij version.

Reply:
:+1: 
i added the test-mutes here to fix that, seems to be working fine.

Reply:
i also tried breaking a test to ensure they are really run :)